pkgs/by-name/on/onthespot/package.nix

```diff
@@ -45,9 +49,11 @@ python3.pkgs.buildPythonApplication rec {
     packaging
     pillow
     protobuf
-    pyqt5
-    pyqt5-sip
+    pyperclip
+    pyqt6
+    pyqt6-sip
     pyxdg
+    qt6.qtbase
     requests
     setuptools
```

Comment:
Is `setuptools` still required in the buildInputs after it was added to `build-system`?

Reply:
No, I'll remove it from buildInputs